hack/release.toml

```diff
@@ -243,6 +243,36 @@ Talos now uses `registry.k8s.io` instead of `k8s.gcr.io` for Kubernetes containe
 See [Kubernetes documentation](https://kubernetes.io/blog/2022/11/28/registry-k8s-io-faster-cheaper-ga/) for additional details.
 
 If using registry mirrors, or in air-gapped installations you may need to update your configuration.
+"""
+
+    [notes.registry-mirrors]
+        title = "Registry Mirrors"
+        description = """\
+Talos had an inconsistency in the way registry mirror endpoints are handled when compared with `containerd` implementation:
+
+```yaml
+machine:
+    registries:
+        mirrors:
+            docker.io:
+                endpoints:
+                    - "https://mirror-registry/v2/mirror.docker.io"
+```
+
+Talos would use endpoint `https://mirror-registry/v2/mirror.docker.io`, while `containerd` would use `https://mirror-registry/v2/mirror.docker.io/v2`.
+This inconsistency is now fixed, and Talos uses same endpoint as `containerd`.
+
+New `overridePath` configuration was introduced to skip appending `/v2` both on Talos and containerd side:
```

Comment:
was introduced means it was there previously, right? I am not grokking this :sweat_smile:

Reply:
how do you say that? `is introduced` ?